azure/template.json

```diff
@@ -172,6 +172,9 @@
                     "subnetResourceId": {
                         "value": "[parameters('sharedBackEndSubnetResourceId')]"
                     },
+                    "netFrameworkVersion": {
+                        "value": "v6.0"
```

Comment:
@CloudPlatformer Please could you confirm if this is needed here as well

Reply:
@CloudPlatformer The pipeline was complaining about this property, so I have reverted this addition